README.md

```diff
@@ -1 +1,44 @@
 
+# Project Name
+
+To Do List Application
+
+# PR
+
+1-Add items to the to do list
+2-Checkbox input form 
+![screenshot](./Capture.PNG)
+
+## Built With
+
+- Html
+- css
+- JavaScript
+
+
+## Getting Started
+
+-open the index.html file in chrome and see the To do list
+
+### Setup
+
+-click the code button in the repo and then download the files as a zip file
+
+#### To get a local copy up and running follow these simple example steps.
+
+- Click the green `Code` button on the repo and copy link
+- In your local PC, open your terminal in the folder you would like to clone the project.
+- Clone the repo with the command: `git clone (copied link)`; like so: `git clone https://github.com/MeqdamAlqudah/To-Do-List`
+- On the terminal, navigate into the directory like so: `cd To-Do-List`
+
```

Comment:
[OPTIONAL]

- I would highly recommend including the steps to install the npm packages and to run the server which are `npm i` and `npm start`. Please include this in the README file before merging so that any user can check your project easily with the given steps.